### PR TITLE
#62 - lpd described

### PR DESCRIPTION
**Pull request by stepanvalyavskiy**
#62 - lpd short description

### CODE REVIEW

File: white-paper/wp.tex [hunk 1]
```diff
@@ -344,12 +344,27 @@ \subsubsection{Discovery}
...
+    and is the space from which an organization should allocate sub-
+    ranges when defining scopes for private use.
+    ff15::efc0:988f also comes from \href{http://tools.ietf.org/html/rfc4291#section-2.7}{IPv6 spec} and means:
+    \begin{itemize}
+      \item[--] FF == Multicast
+      \item[--] 1 == ‘Flags' - where 1 indicates a non-permanently-assigned ("transient” or "dynamically" assigned) multicast address.
+      \item[--] 5 == Site-Local scope
+      \item[--] efc0:988f - the hex representation of 239.192.152.143
+    \end{itemize}
+    We could easily implement BEP 14 as it described and take \href{https://github.com/transmission/transmission/blob/7f147c65fb07a6baed3d079703ff0a31d1b1ca4c/libtransmission/tr-lpd.c}{this implementation} as an example.
```
Comment (g4s8) [inline white-paper/wp.tex:366]: @stepanvalyavskiy please use `\code{}` function to display IP addresses with monospaced font

File: white-paper/wp.tex [hunk 2]
```diff
@@ -344,12 +344,27 @@ \subsubsection{Discovery}
...
     For security reasons, in untrusted networks, the broadcast messages may be optionally signed with
     node private keys to be verified by the receiver using CA public certificate.
-    For optimization reasons, this protocol is resused by metadata exchange protocol for
+    For optimization reasons, this protocol is reused by metadata exchange protocol for
     repository hash table propagation (see \ref{sec:metadata}). \todo{lookup for researches}.
-  \item[Distributed DB] - network addresses could be propagated to the system using supplimentary distributed
+    There are 2 common ways to implement LDP: \href{http://bittorrent.org/beps/bep_0014.html}{bep 14} and \href{http://bittorrent.org/beps/bep_0026.html}{bep 26}.
+    Bep 14 is SSDP-like style and Bep 26 is zeroconf style LDP. To implement Bep 26 each host is required to run a zeroconf service discovery daemon.
+    There is a popular \href{https://github.com/grandcat/zeroconf}{go zeroconf implementation} that could be used.
+    BitTorrent uses following multicast groups: A) 239.192.152.143:6771 (org-local) and B) [ff15::efc0:988f]:6771 (site-local) for Bep 14 implementation.
```
Comment (g4s8) [inline white-paper/wp.tex:355]: @stepanvalyavskiy what does "org-local" and "site-local" mean?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@stepanvalyavskiy please check my comments

File: white-paper/wp.tex (hunk 1 above)
Comment (stepanvalyavskiy) [reply white-paper/wp.tex]: Done

File: white-paper/wp.tex (hunk 2 above)
Comment (stepanvalyavskiy) [reply white-paper/wp.tex]: short description of site & org local and corresponding links to RFC added.

**Review by g4s8 — APPROVED** [review]
@stepanvalyavskiy thanks

**Pull request merged**